elastic/kibana
# [Console] Add import functionality

## Summary

This PR adds import file functionality into the editor. We only allow users to select files of mime type `text/*` to ensure they dont select blobs.

##### How to test

* Start up es and kibana
* Navigate to `Devtools`
* Verify that:
  * Files can be imported into the editor
  * If an empty file is selected an error message is shown


https://github.com/user-attachments/assets/058367a6-d177-4107-ba52-d2efe9010606


<details>
<summary>UI screenshots</summary>

![Screenshot 2024-09-04 at 09 18 19](https://github.com/user-attachments/assets/823b6156-e777-40c7-bbea-6d14c62a98d8)
![Screenshot 2024-09-04 at 09 04 49](https://github.com/user-attachments/assets/106eee34-6207-492a-bb0e-88030f89d771)
![Screenshot 2024-09-04 at 09 05 54](https://github.com/user-attachments/assets/1c8ccf7d-c09c-45fe-9573-844fde5251bb)

</details>
